src/pc.jl

```diff
@@ -203,3 +203,28 @@ function pcalg(n::V, I, par...) where {V}
     end
     dg
 end
+
+"""
+    pcalg(t::T; test=:gausscitest) where{T}
+run PC algorithm for tabular input data t
+"""
+function pcalg(t::T, p::Float64; test=:gausscitest, kwargs...) where{T}
```

Comment:
```suggestion
function pcalg(t, p::Float64; test=:gausscitest, kwargs...)
```
you never use `T`...